contrib/completion/bash/docker

```diff
@@ -688,7 +689,7 @@ __docker_pos_first_nonflag() {
 		(( counter++ ))
 	done
 
-	echo $counter
+	echo "$counter"
```

Comment:
Does shellcheck insist on this?

Reply:
Yes, it did (although a bit odd that it wasn't smart enough to detect that it's not needed 🙄;

```
In contrib/completion/bash/docker line 692:
	echo $counter
             ^------^ SC2086: Double quote to prevent globbing and word splitting.

Did you mean:
	echo "$counter"
```

Reply:
This creates the wrong impression that the output might contain spaces. But even if it did, quoting would not be necessary here.

That said, let's keep it qouted. It's not worth a shellcheck exception.

Reply:
Yes, that was my train of thought as well; this looked less messy than an exception